composer.json

```diff
@@ -13,14 +13,14 @@
         }
     ],
     "require": {
-        "matthiasnoback/symfony-config-test": "^1.0|^2.0",
-        "symfony/dependency-injection": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0",
-        "symfony/yaml": "^2.7|^3.0",
-        "sebastian/exporter": "^1.0|^2.0"
+        "matthiasnoback/symfony-config-test": "^1.0 || ^2.0 || ^3.0",
```

Comment:
Are you sure you can use 3.0 of this library?

Reply:
travis hit me faster then you :-)